infrastructure/terragrunt.hcl

```diff
@@ -90,6 +106,7 @@ inputs = {
   location = get_env("LOCATION")
   naming_suffix = get_env("NAMING_SUFFIX")
   truncated_naming_suffix = get_env("TRUNCATED_NAMING_SUFFIX")
+  deployer_ip_address = get_env("DEPLOYER_IP_ADDRESS", "") // deployer's IP address is added to resource firewall exceptions IF in local_mode
 
```

Comment:
This is where you could add `local_mode` to be passed into each module as a variable

Reply:
gotcha, done